data/presets/presets/place/welcome_sign.json

```diff
@@ -0,0 +1,15 @@
+{
+  "icon": "maki-lighthouse",
+  "geometry": [
+    "point",
+    "area"
+  ],
+  "terms": [
+    "new location"
+  ],
+  "tags": {
+    "description": "A sign welcoming users to a place",
+    "place": "welcome"
+  },
+  "name": "Welcome Sign"
+}
```

Comment:
Can you move this file to `data/presets/presets/tourism/information/board/welcome_sign.json`? We try to keep presets ordered hierarchically like that.